index.d.ts

```diff
@@ -0,0 +1,2 @@
+export interface RandomGenerationError extends Error { }
+export function secureRandomNumber(minimum: number, maximum: number, cb: (err: Error, result: number) => void): void;
```

Comment:
this is two years old, and I'm not sure that the maintainer is particularly active anymore, but in case this ever gets looked at, the signature here needs some work: the `cb` parameter is optional, and a (bluebird) `Promise` is returned.

Reply:
Work like a charm :) 

I hope someone makes types library.. :P

Reply:
Fixed type definitions